frontend/lib/ui/artefact_page/test_execution_pop_over.dart

```diff
@@ -36,10 +36,12 @@ class TestExecutionPopOverState extends ConsumerState<TestExecutionPopOver> {
   Function(bool?)? getOnChangedCheckboxListTileFunction(
     TestExecutionReviewDecision testExecutionReviewDecision,
   ) {
-    if ((testExecutionReviewDecision == TestExecutionReviewDecision.rejected &&
-            (_canReject)) ||
-        (testExecutionReviewDecision != TestExecutionReviewDecision.rejected &&
-            _canApprove)) {
+    if (!testExecutionReviewDecision.isDeprecated &&
+        ((testExecutionReviewDecision == TestExecutionReviewDecision.rejected &&
+                _canReject) ||
+            (testExecutionReviewDecision !=
+                    TestExecutionReviewDecision.rejected &&
+                _canApprove))) {
```

Comment:
can you simplify or make this if statement easier to understand somehow? one suggestion is to just create a variable whose name explains `((testExecutionReviewDecision == TestExecutionReviewDecision.rejected &&
                _canReject) ||
            (testExecutionReviewDecision !=
                    TestExecutionReviewDecision.rejected &&
                _canApprove))`

Reply:
I moved the second part in the new variable as suggested, and added a comment that explains what is happening here.